modules/users/client/controllers/signup.client.controller.js

```diff
@@ -21,7 +21,7 @@
     vm.suggestionsLimit = 3; // How many tribes suggested (including possible referred tribe)
 
     // Variables for invitation feature
-    vm.invitationCode = '';
+    vm.invitationCode = $location.search().code;
```

Comment:
You can use `$stateParams.code` here, and should probably be `$stateParams.code || ''` just to keep it as a string.